readme.md

```diff
@@ -31,3 +31,42 @@ jsdoc(schema);
   * @property {integer} [age] - A person's age
   */
 ```
+
+### Options
+
+- `autoDescribe` - Boolean on whether to add a description
```

Comment:
This is great, thanks! I'm thinking how about including the type earlier, ala TypeScript? Something like `autoDescribe: boolean` is that more readable?

Reply:
Sure, sounds good. Added a commit just now if you want to take a look.

Reply:
Looks good. Once we've sort out the other PR and whether there's a case for emptyDescriptionLineBreak and propertiesLineBreak this can go next.